scripts/enable-datastore-viewer.sh

```diff
@@ -96,12 +100,17 @@ server {
     }
 }
 "
-	if [ -e /etc/nginx/sites-enabled/${app_id}.conf ]; then
-		cp /etc/nginx/sites-enabled/${app_id}.conf /tmp
-		echo "$pippo" >> /etc/nginx/sites-enabled/${app_id}.conf
-		echo "Datastore viewer enabled for $app_id at http://$(cat /etc/appscale/my_public_ip):$VIEWER_PORT. Allowed IP: $IP."
-		service nginx reload
-	else
-		echo "Cannot find configuration for ${app_id}."
-	fi
-done
+
+if [ -e /etc/nginx/sites-enabled/${APP_ID}.conf ]; then
+        echo "$config" > /etc/nginx/sites-enabled/${APP_ID}_datastore_viewer.conf
+        echo "Datastore viewer enabled for ${APP_ID} at http://$(cat /etc/appscale/my_public_ip):${VIEWER_PORT}. Allowed IP: $IP."
+        service nginx reload
```

Comment:
You may want to print the success statement after the reload in case the reload fails.